docs/ref/forms/api.txt

```diff
@@ -1448,7 +1477,10 @@ If you need to access some additional information about a form field in a
 template and using a subclass of :class:`~django.forms.Field` isn't
 sufficient, consider also customizing :class:`~django.forms.BoundField`.
 
-A custom form field can override ``get_bound_field()``:
+A custom form can override :attr:`~django.forms.Form.bound_field_class` and
+a custom form field takes an optionnal
+:attr:`~django.forms.Field.bound_field_class` constructor argument.
+ Additionnally, it can override ``get_bound_field()``:
```

Comment:
```suggestion
a custom form field takes an optional
:attr:`~django.forms.Field.bound_field_class` constructor argument.
 Additionally, it can override ``get_bound_field()``:
```

Reply:
Yes, the CI pointed me that. I just fixed this.